polkadot/node/core/runtime-api/src/lib.rs

```diff
@@ -612,5 +622,12 @@ where
 			sender,
 			result = (index)
 		),
+		Request::BulkCoreCount(index, sender) => query!(
```

Comment:
Why do we need a runtime API for this? The node should not care at all about the number of bulk cores.

Reply:
Gotcha, that makes sense.